components/board/customlinux/README.md

```diff
@@ -15,4 +15,4 @@ This component supports a board running Linux and requires the user to provide a
 ```
 
 `pwm_chip_sysfs_dir` and `pwm_id` are optional fields.
-To configure a new board with these mappings, set the `pin_config_file_path` attribute to the filepath to your json configuration file.
\ No newline at end of file
+To configure a new board with these mappings, set the `pin_config_file_path` attribute to the filepath to your json configuration file.
```

Comment:
I don't see the diff here. I suspect Natalie's version did not have a newline at the end of the file, which it should for POSIX compliance, but which half the text editors out there don't add by default. :shrug: